core/tests/test_utils.py

```diff
@@ -2118,7 +2118,7 @@ def set_collection_editors(self, collection_editor_usernames):
         for name in collection_editor_usernames:
             self.add_user_role(name, feconf.ROLE_ID_COLLECTION_EDITOR)
 
-    def get_user_id_from_email(self, email):
+    def get_user_id_from_email(self, email) -> str:
```

Comment:
not done

Reply:
@U8NWXD i think that my previous comment wasn't visible to you

Reply:
No, I don't think so. The fact that the mypy checks are failing suggests that those other files assume that `get_user_id_from_email` always returns a string, which is not the case since `get_user_settings_by_auth_id` can return `None`:

https://github.com/oppia/oppia/blob/49a9c2087cb53c8b22b10748b93ab1769edfb8dd/core/domain/user_services.py#L274-L302

I definitely don't think we should add the incorrect `-> str` type hint. However, if it's okay with @nithusha21, I would be okay with adding mypy ignore comments in this case since fixing the types would be a lot of work that's out of scope for your task of just adding backend tests.

Reply:
@U8NWXD should i proceed to remove the `-> str` annotation ? and does it another issue to be resolved ?

Reply:
Yes, I think it's fine to remove the `-> str` annotation. I don't think you need to open an issue for this though--it should be fixed as part of #14033.

Reply:
done